From: Kay Sievers <kay.sievers@vrfy.org>
Date: Wed, 18 Mar 2009 01:32:12 +0000 (+0100)
Subject: rules: fix extra quote in 50-udev-default.rules
X-Git-Url: http://git.kernel.org/?p=linux%2Fhotplug%2Fudev.git;a=commitdiff_plain;h=3b8103d45f99218f44b4ea87ab4284f6a253409b

rules: fix extra quote in 50-udev-default.rules

Patch from Gerardo Exequiel Pozzi.
---

diff --git a/rules/rules.d/50-udev-default.rules b/rules/rules.d/50-udev-default.rules
index 38b954d..240cb3b 100644
--- a/rules/rules.d/50-udev-default.rules
+++ b/rules/rules.d/50-udev-default.rules
@@ -107,7 +107,7 @@ KERNEL=="hw_random",		NAME="hwrng"
 KERNEL=="mmtimer",		MODE="0644"
 KERNEL=="rflash[0-9]*",		MODE="0400"
 KERNEL=="rrom[0-9]*",		MODE="0400"
-KERNEL=="sxctl",		NAME="specialix_sxctl""
+KERNEL=="sxctl",		NAME="specialix_sxctl"
 KERNEL=="rioctl",		NAME="specialix_rioctl"
 KERNEL=="iowarrior[0-9]*",	NAME="usb/%k"
 KERNEL=="hiddev[0-9]*",		NAME="usb/%k"
